src/items/class-features/invisibility_field_(ex_drone).json

```diff
@@ -44,7 +44,7 @@
       "gmnotes": "",
       "short": "",
       "unidentified": "",
-      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.cQtmz297P5g6g5xv]{Advanced Mods}</p>\n<p>Once per day as a standard action, your drone can turn invisible for 10 minutes, as per the <em>invisibility</em> spell. If it makes an attack during this time, the invisibility ends. Your drone can spend 2 Resolve Points to use this ability again on the same day. Your drone must have the reactive camouflage mod to select this.</p>\n<p>@UUID[Compendium.sfrpg.class-features.ZCNA7qg5Hz6Jirs0]{Reactive Camouflage (Ex; Drone)}</p>\n<p>@UUID[Compendium.sfrpg.spells.YDXegEus8p0BnsH1]{Invisibility}</p>"
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.cQtmz297P5g6g5xv]{Advanced Mods}</p>\n<p>Once per day as a standard action, your drone can turn invisible for 10 minutes, as per the <em>invisibility</em> spell @UUID[Compendium.sfrpg.spells.YDXegEus8p0BnsH1]{Invisibility}. If it makes an attack during this time, the invisibility ends. Your drone can spend 2 Resolve Points to use this ability again on the same day. Your drone must have the reactive camouflage mod @UUID[Compendium.sfrpg.class-features.ZCNA7qg5Hz6Jirs0]{Reactive Camouflage (Ex; Drone)} to select this.</p>"
```

Comment:
For the inline links, replace the original word with the link
e.g `as per the <em>@UUID[Compendium.sfrpg.spells.YDXegEus8p0BnsH1]{invisibility}</em> spell`

Reply:
Will fix the inline links.

If you could tell me how not to include the .db files, I'll try to comply.

Reply:
In VSCode, I usually just revert any changes to .db files after running `cook`, but alternatively you can stage everything you want to commit, which will leave out the `.db`s.

I think VSCode might even let you just "destage" the dbs and it'll automatically stage everything else.

Reply:
So, my current steps after finishing the changes and closing foundry, in VSCode:
npm run unpack
npm run cook
Then I stage all the changes using '+'
Commit the Changes
Sync the branch to Git

Do I need to selectively stage?

Reply:
That way works, just make sure you unstage all dbs before committing (you can unstage by folder, so you can just unstage the entire `packs` folder)